circle/src/deep_quotient.rs

```diff
@@ -30,8 +30,8 @@ pub(crate) fn deep_quotient_vanishing_part<F: ComplexExtendable, EF: ExtensionFi
 
 pub(crate) fn deep_quotient_reduce_row<F: ComplexExtendable, EF: ExtensionField<F>>(
     alpha: EF,
-    x: Complex<F>,
-    zeta: Complex<EF>,
+    x: Point<F>,
+    zeta: Point<EF>,
```

Comment:
What is the use of the `deep_quotient_reduce_row` if we have the full "across evaluation domain" variant below?